src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -624,6 +635,7 @@ public boolean equals(Object o) {
         KubernetesCloud that = (KubernetesCloud) o;
         return skipTlsVerify == that.skipTlsVerify &&
                 addMasterProxyEnvVars == that.addMasterProxyEnvVars &&
+                ensureHomeIsValid == that.ensureHomeIsValid &&
```

Comment:
```
[ERROR] Suspicious comparison of Boolean references in org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud.equals(Object) [org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud] At KubernetesCloud.java:[line 636] RC_REF_COMPARISON_BAD_PRACTICE_BOOLEAN
```

Reply:
I changed the type of that from `boolean` to `Boolean` quite late, so I missed that... Will take a look.